composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^5.4||^7.0",
+        "php": "^7.2",
```

Comment:
FYI This will be reverted before merging this to get this fix out to all current users before raising the package to 7.2

Reply:
This has been reverted